components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -366,51 +372,56 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
     }
 
     async ensureWorkspaceAuth(instanceID: string, retry: boolean) {
-        if (!document.cookie.includes(`${instanceID}_owner_`)) {
-            let attempt = 0;
-            let fetchError: Error | undefined = undefined;
-            while (attempt <= 10) {
-                attempt++;
-
-                let code: number | undefined = undefined;
-                fetchError = undefined;
-                try {
-                    const authURL = gitpodHostUrl.asWorkspaceAuth(instanceID);
-                    const response = await fetch(authURL.toString());
-                    code = response.status;
-                } catch (err) {
-                    fetchError = err;
-                }
+        if (document.cookie.includes(`${instanceID}_owner_`)) {
+            // Cookie already present
+            return;
+        }
 
-                if (retry && (code === 404 || !!fetchError)) {
-                    console.warn("Unable to retrieve workspace-auth cookie! Retrying shortly...", fetchError, {
-                        code,
-                        attempt,
-                    });
-                    // If the token is not there, we assume it will appear, soon: Retry a couple of times.
-                    await new Promise((resolve) => setTimeout(resolve, 2000));
-                    continue;
-                }
-                if (code !== 200) {
-                    // getting workspace auth didn't work as planned
-                    console.error("Unable to retrieve workspace-auth cookie! Quitting.", {
-                        code,
-                        attempt,
-                    });
-                    return;
-                }
+        const MAX_ATTEMPTS = 10;
+        const ATTEMPT_INTERVAL = 2000; // [ms]
```

Comment:
```suggestion
        const ATTEMPT_INTERVAL_MS = 2000;
```
Makes it easier when reading the var down below, without having to refer back to the declaration.